test/xpu_api/numerics/complex.number/cases.h

```diff
@@ -218,48 +215,51 @@ CLANG_DIAGNOSTIC_IGNORED_AUTOLOGICAL_CONSTANT_COMPARE
         return non_zero_nan;
     }
     return non_zero;
-
-CLANG_DIAGNOSTIC_POP
 }
 
 inline
 int
 classify(double x)
 {
-// Suppress clang warning: comparison with infinity always evaluates to false in fast floating point modes [-Wtautological-constant-compare]
-CLANG_DIAGNOSTIC_PUSH
-CLANG_DIAGNOSTIC_IGNORED_AUTOLOGICAL_CONSTANT_COMPARE
-
     if (x == 0)
         return zero;
     if (std::isinf(x))
         return inf;
     if (std::isnan(x))
         return NaN;
     return non_zero;
-
-CLANG_DIAGNOSTIC_POP
 }
 
-void is_about(float x, float y)
+auto
+get_eps(float)
 {
-// Suppress clang warning: floating-point comparison is always true; constant cannot be represented exactly in type 'float' [-Wliteral-range]
-CLANG_DIAGNOSTIC_PUSH
-CLANG_DIAGNOSTIC_IGNORED_LITERAL_RANGE
-
-    assert(std::abs((x-y)/(x+y)) < 1.e-6);
+    constexpr float eps = 1.e-6;
+    return eps;
+};
 
-CLANG_DIAGNOSTIC_POP
-}
+template <typename T>
+typename std::enable_if<!std::numeric_limits<T>::is_integer, T>::type
+get_eps(T)
+{
+    static_assert(::std::is_same_v<T, double> || ::std::is_same_v<T, long double>);
+    constexpr T eps = 1.e-11;
```

Comment:
why 1e-14 was changed by 1e-11?

Reply:
For example, in source implementation in test pow.complex.pass we check source data:
```assert(std::fabs((x - y) / (x + y)) < exp);```
where
```x == 57.004343176752272```
```y == 57.004343176752229```

In this case by using old formula we had:
```std::fabs((x - y) / (x + y)) == 3.73941368058711395217e-16```

By using new formula we have:
```std::fabs(x - y) == 4.26325641456060111523e-14```

Does this mean that our results is not good enough? I think no.
I think that this mean that old epsilon constants is simple too small for our new formula.

This is the reason why these constant's were changed.